backport-changelog/7.2/13399.md

```
@@ -0,0 +1,3 @@
+https://github.com/WordPress/wordpress-develop/pull/13399
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)